Source/UI/NewUI/BillingForm/Default Implementation/BillingForm/Header/Cancel/DefaultCancelButtonFormStyle.swift

```diff
@@ -12,7 +14,9 @@ struct DefaultCancelButtonFormStyle: CKOButtonStyle {
     var height: Double
     var width: Double
     
-    init( isEnabled: Bool = true,
+    init( textColor: UIColor = .clear,
+          isHidden: Bool = false,
+          isEnabled: Bool = true,
```

Comment:
Similarly to the properties. In this case this can be dismissed as its not public, but it would be a good habit to spend 2 minutes looking at how it would look at call site.

Personally I'd say that
```
DefaultCancelButtonFormStyle(textColor: ... , isHidden: ... , isEnabled: ... , text: ... , font: ... ....)
```
could be improved. But again, not a public facing, just a good exercise to improve how our code is perceived.